instant-clip-tokenizer-py/src/lib.rs

```diff
@@ -0,0 +1,135 @@
+use std::fs::File;
+use std::io::BufReader;
+
+use numpy::{IntoPyArray, PyArray2};
+use pyo3::exceptions::PyValueError;
+use pyo3::prelude::*;
+
+#[pymodule]
+#[pyo3(name = "instant_clip_tokenizer")]
+fn instant_clip_tokenizer_py(_py: Python, m: &PyModule) -> PyResult<()> {
+    m.add_class::<Tokenizer>()?;
+    Ok(())
+}
+
+/// A text tokenizer for the CLIP neural network.
+#[pyclass]
+struct Tokenizer {
+    inner: instant_clip_tokenizer::Tokenizer,
+}
+
+#[pymethods]
+impl Tokenizer {
+    /// Create a new `Tokenizer` using the vocabulary data bundled with this
+    /// library.
+    ///
+    /// The resulting `Tokenizer` is suitable for use with the original CLIP
+    /// model.
+    #[new]
+    fn new() -> Self {
```

Comment:
What's the point of the `new()` method? How could it be used in practice?

Reply:
It's the constructor of `Tokenizer` - do you want me to add an example to the doc-comment? Not sure I understand your questions...

Regarding doc-comments for the bindings: I intentionally did not put any examples in here because they will end up as documentation in the generated Python library, so the examples should be Python code, not Rust.
However, having Python examples in a Rust crate feels wrong... so I'm thinking, instead, we should maybe just provide an example in the Readme that shows the major use-case?

Reply:
Ah never mind, I got confused about `new()` using the built-in vocabulary.

Providing an example for the Python bindings in the README sounds good!